backend/src/main/java/com/google/sps/notifiers/EmailNotifier.java

```diff
@@ -0,0 +1,49 @@
+// Class used to send users emails regarding their status in regards to their Ad-Lib Meeting.
+
+import com.sendgrid.*;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.*;
+
+import java.io.IOException;
+
+
+
+public class EmailNotifier extends Notifier {
+
+    // Member Variables 
+    
+    public String toEmail ;       
+    public String fromEmail;
+
+    public EmailNotifier(String Name, String fromEmail, String toEmail)
+    {
+
+        super(Name);
+        this.toEmail = toEmail;
+        this.fromEmail = fromEmail;
+    }
+    
+    public static void main(Strings[], args) throws IOException {
```

Comment:
Can you elaborate on how this EmailNotifier class should be used? I expect that some other part of the backend code will call into it to send emails which would mean that the body of this method should instead be an instance method on this class. public static void main will only be called if this file were run as a standalone java program.

Reply:
i was using the main function to test functionality. It will be used via a send email function

Reply:
Understood. Let's take it out before submitting the PR though as it won't be used in production